rcode-reader/src/main/java/eu/rssw/pct/elements/VariableElement.java

```diff
@@ -0,0 +1,78 @@
+package eu.rssw.pct.elements;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.util.Set;
+
+import eu.rssw.pct.AccessType;
+import eu.rssw.pct.AccessibleClassElement;
+import eu.rssw.pct.DataType;
+import eu.rssw.pct.RCodeInfo;
+
+public class VariableElement extends AccessibleClassElement {
+  private static final int READ_ONLY = 1;
+  private static final int WRITE_ONLY = 2;
+  private static final int BASE_IS_DOTNET = 4;
+  private static final int NO_UNDO = 8;
+
+  private int data_type;
+  private int recordPosition;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "recordPosition" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS1068)